tests/unit/controllers/index-test.js

```diff
@@ -140,18 +151,19 @@ module('Unit | Controller | index', function(hooks) {
 
       {
         handle: '@zoey',
+        name: 'Zoey',
         profileLink: 'https://github.com/zoey'
       },
     ]);
 
     assert.strictEqual(
       controller.contributorsList,
       [
-        '<a href="https://github.com/emberjs" rel="noopener noreferrer" target="_blank">@emberjs</a>',
-        '<a href="https://github.com/tomster" rel="noopener noreferrer" target="_blank">@tomster</a>',
-        '<a href="https://github.com/zoey" rel="noopener noreferrer" target="_blank">@zoey</a>'
+        '<a href="https://github.com/emberjs" rel="noopener noreferrer" target="_blank">(@emberjs)</a>',
+        '<a href="https://github.com/tomster" rel="noopener noreferrer" target="_blank">(@tomster)</a>',
```

Comment:
If we decide to not use parentheses when `name` isn't present, these lines will need to be updated.